src/main/java/io/xpring/Utils.java

```diff
@@ -68,6 +68,37 @@ public static String byteArrayToHex(byte [] bytes) {
         return data;
     }
 
+    /**
+     * Encode the given classic address and tag into an X-Address.
+     *
+     * @param classicAddress A classic address to encode.
+     * @param tag An optional tag to encode.
+     * @return A new X-Address if inputs were valid, otherwise undefined.
+     * @see <a href="https://xrpaddress.info/">https://xrpaddress.info/</a>
+     */
+    public static String encodeXAddress(String classicAddress, Long tag) {
```

Comment:
Is there a reason to not accept an instance of `ClassicAddress` as the argument here instead of its decomposed fields?